services/common/src/redux/reducers/rootReducerShared.ts

```diff
@@ -50,6 +50,7 @@ import permitConditionDiffReducer, { permitConditionDiffReducerType } from "../s
 const networkReducers = Object.fromEntries(Object.entries(NetworkReducerTypes).map(([key, value]) =>
   [NetworkReducerTypes[key], createReducer(networkReducer, value)]
 ));
+import permitSearchReducer, { permitSearchReducerType } from "../slices/permitSearchSlice";
```

Comment:
this is a nitpicky thing, but can we put this import with the other imports? It's below a const declaration and that irks me. 😂

Reply:
I should really install an auto-formatter of imports on save 😆

Reply:
I think my auto-formatter broke at some point. 😭 
Suddenly I see that PRs that changed the files I did also changed my formatting, it didn't used to!

This one is weird, I've noticed it doing it before in the same place. Like if you just import your stuff it adds it underneath the const and it bothers me.